specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/compute.json

```diff
@@ -187,10 +187,7 @@
         ],
         "responses": {
           "200": {
-            "description": "OK",
-            "schema": {
-              "$ref": "#/definitions/OperationStatusResponse"
-            }
```

Comment:
Why are we removing the schema? If this is removed, what will be schema of the response?

Reply:
For Delete or POST operation, the body of response is empty.  So the Swagger spec is wrong, and we get nothing for OperationStatusResponse anyway.  So I removed it.  

https://github.com/Azure/azure-sdk-for-net/blob/6a493f405a59f7ea4f984143d5afe39ddeb8a8dd/src/SDKs/Compute/Compute.Tests/SessionRecords/Compute.Tests.AvailabilitySetTests/TestOperations.json#L747